api/auth.py

```
@@ -483,4 +483,98 @@ async def delete_api_key(
     db.delete(api_key)
     db.commit()
     
-    return {"message": "API key deleted successfully"}
\ No newline at end of file
+    return {"message": "API key deleted successfully"}
+
+# Admin/Testing endpoints
+@router.post("/create-test-user")
+async def create_test_user(
+    admin_key: str,
+    db: Session = Depends(get_db)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)